tests/test_gauss_pdf_and_cdf.py

```diff
@@ -0,0 +1,30 @@
+from os.path import exists
+from src.gauss_pdf_and_cdf import (GaussianProbabilisticSetting,
+                                   BetaProbabilisticSetting)
+import cv2
+
+
+def show_or_save_figure(distribution_type):
+    if distribution_type == "gaussian":
+        distribution = GaussianProbabilisticSetting()
+    elif distribution_type == "beta":
+        distribution = BetaProbabilisticSetting()
+
+    lengths = distribution.show_figure()
+
+    for length in lengths:
+        assert length[0] == length[1], 'length should be same'
```

Comment:
What does length mean?

Reply:
length[0], length[1] is x.length, y.length in figures.
But I don't know how to test the show_figure function correctly.

Reply:
You must show its meaning with their names and your current error message is unclear.

Reply:
You can't assume people will ask you questions. Just improve your code.